tapeout/src/main/scala/transforms/Generate.scala

```diff
@@ -125,23 +126,23 @@ trait HasTapeoutOptions { self: ExecutionOptionsManager with HasFirrtlOptions =>
         harnessConf = Some(x)
       )
     }.text {
-      "use this to set the harness conf file location"
-    }
+    "use this to set the harness conf file location"
+  }
 
 }
 
 case class TapeoutOptions(
-  harnessOutput: Option[String] = None,
-  synTop: Option[String] = None,
-  topFir: Option[String] = None,
-  topAnnoOut: Option[String] = None,
-  topDotfOut: Option[String] = None,
-  harnessTop: Option[String] = None,
-  harnessFir: Option[String] = None,
-  harnessAnnoOut: Option[String] = None,
-  harnessDotfOut: Option[String] = None,
-  harnessConf: Option[String] = None
-) extends LazyLogging
+                           harnessOutput: Option[String] = None,
+                           synTop: Option[String] = None,
+                           topFir: Option[String] = None,
+                           topAnnoOut: Option[String] = None,
+                           topDotfOut: Option[String] = None,
+                           harnessTop: Option[String] = None,
+                           harnessFir: Option[String] = None,
+                           harnessAnnoOut: Option[String] = None,
+                           harnessDotfOut: Option[String] = None,
+                           harnessConf: Option[String] = None
+                         ) extends LazyLogging
```

Comment:
Please revert this style change

Reply:
I believe this is a default IntelliJ style that you can and should (IMO) disable